kong/tools/aws_stream.lua

```diff
@@ -0,0 +1,181 @@
+--- Stream class.
+-- Decodes AWS response-stream types, currently application/vnd.amazon.eventstream
```

Comment:
Does this stream parse functionality need to be implemented in `lua-resty-aws` library? it seems not Kong's scope

Reply:
We have another [ticket](https://konghq.atlassian.net/browse/KAG-3834)/[PR](https://github.com/Kong/lua-resty-aws/pull/119) tracking this effort since it requires more work being done than it looks like. I think it should be good to just put it here for now. We can remove it and call SDK in the future after that work finished

Reply:
Yes I wrote this for lua-resty-aws originally, and it's in the PR, but we have run out of time to re-test all AWS interfaces.

We told it was easier to just put in here, and we'll use the SDK after it's fixed.